Example/Datasnap/DSIOAppDelegate.m

```diff
@@ -25,8 +25,10 @@ - (BOOL)application:(UIApplication*)application didFinishLaunchingWithOptions:(N
                            apiKeySecret:@"KA0HdzrZzNjvUq8OnKQoxaReyUayZY0ckNYoMZURxK8"
                          organizationId:@"19CYxNMSQvfnnMf1QS4b3Z"
                               projectId:@"21213f8b-8341-4ef3-a6b8-ed0f84945186"
-                         eventQueueSize:1
+                              idfaOptIn:YES
+                                  email:@"amcintyre@grio.com"
```

Comment:
I see this is in an example directory, but I would probably use a dummy email 'someEmail@example.com' or something of the sort